README.md

```diff
@@ -1,11 +1,15 @@
-## En Français ?
+## ¿En Español?
 
-Clique sur <a href="README.fr.md">ce lien</a> pour accéder à la version française du setup.
+Haga clic en <a href="README.es.md">este enlace</a> para acceder a la versión en español de la configuración.
```

Comment:
@lucasgday @sbuffose I've added this sentence in the README so that students can switch to the Spanish setup from the home page. Don't hesitate to update the wording if my Spanish is not correct :pray:

Reply:
@dmilon it's perfect! Thanks!